napalm_eos/eos.py

```diff
@@ -985,16 +985,20 @@ def get_mac_address_table(self):
     def get_route_to(self, destination='', protocol=''):
         routes = dict()
 
+        if protocol.lower() not in ['', 'bgp', 'connected', 'isis', 'ospf', 'rip', 'static']:
```

Comment:
Eeerm... I know I'll contradict ourselves (did we discuss that before?) but I think we should remove this line completely. I don't see any reason why we'd limit to some certain protocols only. I will make the necessary adjustment on the other drivers in order to avoid this limitation!